ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Ledger.hs

```diff
@@ -488,9 +485,9 @@ getFilteredDelegationsAndRewardAccounts ss creds =
   Serialisation
 -------------------------------------------------------------------------------}
 
-serialisationFormatVersion0 :: VersionNumber
-serialisationFormatVersion0 = 0
-
+-- | 'serialisationFormatVersion0' used to include the 'LedgerViewHistory', but
```

Comment:
Use `@` instead of `'` to prevent these Haddock references to `*Version0` from emitting warnings.